opentelemetry-api/src/opentelemetry/util/types.py

```diff
@@ -56,3 +56,5 @@
     ],
     ...,
 ]
+
+LogAttributes = Mapping[str, "AnyValue"]
```

Comment:
Based on the last discussions in Spec and Log SIG, calling the `LogAttributes` may be problematic since we may eventually support complex attributes on events and spans. So we should consider calling them something signal-agnostic (`ComplexAttributes`, `ExtendedAttributes`, etc). 

I'd suggest to hold this PR back for a bit - I'll bring it up on the spec call and let's try to pick a consistent name across different languages.